index.html

```diff
@@ -188,6 +210,7 @@ <h5>Skills</h5>
     <!--forms section-->
 
     <section class="form-container">
+        <div class="form-flex">
       <h2 class="form-intro">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </h2>
```

Comment:
Kindly align the left text with the Top of the Contact form as shown in Figma Design 😃

Your Design:
![image](https://user-images.githubusercontent.com/73714615/186674544-8ed09214-c58f-4057-925f-249ecc57abec.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/186643497-0b76c1ee-c631-4625-be2d-d19f56649b34.png)